src/test/java/seedu/address/model/module/UniqueClassListTest.java

```diff
@@ -1,18 +1,26 @@
 package seedu.address.model.module;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.address.testutil.Assert.assertThrows;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.module.exceptions.ClassNotFoundException;
 import seedu.address.model.module.exceptions.DuplicateClassException;
 
+import java.sql.Array;
```

Comment:
Can I check what the sql array is used for/is it an error?

Reply:
Oops Probably imported this by accident haha
Thanks for pointing out!